site/content/docs/5.0/migration.md

```diff
@@ -192,6 +193,7 @@ Badges were overhauled to better differentiate themselves from buttons and to be
 - **Todo:** Drop `.text-hide` as it's an antiquated method for hiding text that shouldn't be used anymore
 - **Todo:** Split utilities into property-value utility classes and helpers
 - Negative margin utilities are disabled by default. You can re-enable them by setting `$enable-negative-margins: true`, but keep in mind this can increase the file size quite a lot.
+- Renamed `.sr-only` and `.sr-only-focusable` to `.v-hidden` and `.v-hidden-focusable`.
```

Comment:
Let's also repeat this in a `## Alpha 2` section at the top of the page. We'll want to highlight changes between alphas.

Reply:
good idea. i took inspiration from the way the v4 migration was structured/split up. Hope this makes sense @mdo ?